packages/e2e/src/page-objects/contacts.page.ts

```diff
@@ -7,122 +7,144 @@ import { ChainablePromiseElement } from "webdriverio"
 import Page from "./page"
 
 class ContactsPage extends Page {
+  /**[Selector] New contact button on Contacts screen */
```

Comment:
I don't think those comments are necessary. There is a getter 'newContantButton', which is part of ContactsPage class :)

The same for other comments :)